ClaimOwnership/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Context } from "@azure/functions";
+import createAzureFunctionHandler from "@pagopa/express-azure-functions/dist/src/createAzureFunctionsHandler";
+import { setAppContext } from "@pagopa/io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import * as express from "express";
+import { getConfigOrThrow } from "../utils/config";
+import getPool from "../utils/db";
+import ClaimOwnershipHandler from "./handler";
+
+const config = getConfigOrThrow();
+const client = getPool(config);
+const setupExpress = (): express.Express => {
+  const app = express();
+  app.post(
+    "/api/v1/organizations/:organizationFiscalCode/:delegate_id",
```

Comment:
Please review this path, I'm not sure it's correct

Reply:
Yep, it's fix by a4cb626